### ISSUE TITLE
[Request] Customize name of Replicache tables

### ISSUE THREAD

**Issue by user72356** [rocicorp/replicache-nextjs#21]
It would be great if we could easily provide custom names for the Replicache internal tables. For instance the "space" table clashes with a table name I'm already using. Right now I'm forking the lib just so I can change the hardcoded table names. 

Thank you!

**Comment by user72356** [comment]
In the meantime maybe add a prefix to the Replicache tables such as "repli_space"?

**Comment by aboodman** [comment]
I agree that it makes sense to prefix the tables with e.g., `replicache_`. Would you be open to providing a PR for this.

**Comment by nbarraud** [comment]
Same issue here and I have 2 PRs coming your way for this (one for the nextjs backend and one for express).
